engine/table/src/test/java/io/deephaven/engine/util/scripts/TestGroovyDeephavenSession.java

```diff
@@ -104,5 +124,486 @@ public void testUnloadedWildcardPackageImport() {
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final int[] var2 = t.getColumnSource("Z", int[].class).get(0);
+            assertArrayEquals(Sort.sort(Y, z), var2);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        // TODO (deephaven-core#4570) sortObj fails with mixed number types
+        // c = "t = emptyTable(1).updateView(\"Y=" + Y
+        // + "\", \"Z=io.deephaven.function.Sort.<Comparable>sortObj(new Comparable[]{Y, d})\")\n";
+        // try {
+        // QueryScope.addParam("z", z);
+        // QueryScope.addParam("d", d);
+        // session.evaluateScript(c).throwIfError();
+        // final Table t = (Table) session.getVariable("t");
+        // final Comparable[] var2 = t.getColumnSource("Z", Comparable[].class).get(0);
+        // // noinspection unchecked
+        // assertArrayEquals(Sort.<Comparable>sortObj(Y, d), var2);
+        // } catch (Exception e) {
+        // e.printStackTrace();
+        // fail("Fail for : \n" + c);
+        // }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=sortDescending(Y, z)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final int[] var2 = t.getColumnSource("Z", int[].class).get(0);
+            assertArrayEquals(Sort.sortDescending(Y, z), var2);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        // TODO (deephaven-core#4570) sortObj fails with mixed number types
+        // c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=sortDescending(new Number[]{Y, d})\")\n";
+        // try {
+        // QueryScope.addParam("z", z);
+        // QueryScope.addParam("d", d);
+        // session.evaluateScript(c).throwIfError();
+        // final Table t = (Table) session.getVariable("t");
+        // final Number[] var2 = t.getColumnSource("Z", Number[].class).get(0);
+        // // noinspection unchecked
+        // assertArrayEquals(Sort.<Comparable>sortDescendingObj(Y, d), var2);
+        // } catch (Exception e) {
+        // e.printStackTrace();
+        // fail("Fail for : \n" + c);
+        // }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=ssVec(Y, z)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+    }
+
+
+    @Test
+    public void testMinInFormula() {
+        QueryScope.addParam("d", 5d);
+        session.evaluateScript("t = emptyTable(1).updateView(\"Y=1\", \"Z=min(Y,d)\")\n").throwIfError();
+        final Table t = (Table) session.getVariable("t");
+    }
```

Comment:
Do import aliases work now? Should we have a test for that?

Reply:
Yes, they do, will add a test.